HISTORY.md

```diff
@@ -1,5 +1,50 @@
 # History
 
+## 1.3.0 - 2023-01-18
+
+This release makes changes to the way that individual transformers are stored in the `HyperTransformer`. When accessing the config via `HyperTransformer.get_config()`, the transformers listed in the config are the actual transformer instances used during fitting and transforming. These instances can now be accessed and used to examine their properties post fitting. For example, you can now view the mapping for a `PseudoAnonymizedFaker` instance using `PseudoAnonymizedFaker.get_mapping()` on the instance retrieved from the config.
```

Comment:
the transformers listed in the config are **now** the actual transformer instances ?